internal/kubernetes/operator/config_exporter_test.go

```diff
@@ -416,3 +416,230 @@ func TestExportAtlasStreamProcessing(t *testing.T) {
 		)
 	})
 }
+func setupMockController(t *testing.T) (*gomock.Controller, *mocks.MockOperatorGenericStore, *ConfigExporter) {
+	t.Helper()
+	ctl := gomock.NewController(t)
+	atlasOperatorGenericStore := mocks.NewMockOperatorGenericStore(ctl)
+	ce := NewConfigExporter(atlasOperatorGenericStore, nil, projectID, orgID).
+		WithTargetNamespace("test").
+		WithTargetOperatorVersion("2.3.0").
+		WithSecretsData(true)
+	return ctl, atlasOperatorGenericStore, ce
+}
```

Comment:
This might be style but this helper mixes too much stuff to my taste, compare to this:

```go
func defaultTestConfigExporter(t *testing.T, genStore *mocks.MockOperatorGenericStore) *ConfigExporter {
    t.Helper()
    return  NewConfigExporter(atlasOperatorGenericStore, nil, projectID, orgID).
		WithTargetNamespace("test").
		WithTargetOperatorVersion("2.3.0").
		WithSecretsData(true)
}
```

Usage is:
```go
	ctl := gomock.NewController(t)
	atlasOperatorGenericStore := mocks.NewMockOperatorGenericStore(ctl)
	ce := defaultTestConfigExporter(t, atlasOperatorGenericStore)
```
Still not a lot of lines and cleared single returns.